VocaDb.ResXFileCodeGenerator/StringBuilderGenerator.cs

```diff
@@ -3,15 +3,33 @@
 using System.Text;
 using System.Text.RegularExpressions;
 using System.Web;
+using System.Xml;
 using System.Xml.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
 
 namespace VocaDb.ResXFileCodeGenerator;
 
 public sealed class StringBuilderGenerator : IGenerator
 {
 	private static readonly Regex ValidMemberNamePattern = new(@"^[\p{L}\p{Nl}_][\p{Cf}\p{L}\p{Mc}\p{Mn}\p{Nd}\p{Nl}\p{Pc}]*$", RegexOptions.Compiled | RegexOptions.CultureInvariant);
 	private static readonly Regex InvalidMemberNameSymbols = new(@"[^\p{Cf}\p{L}\p{Mc}\p{Mn}\p{Nd}\p{Nl}\p{Pc}]", RegexOptions.Compiled | RegexOptions.CultureInvariant);
-	public string Generate(Stream resxStream, GeneratorOptions options)
+
+	private static readonly DiagnosticDescriptor DuplicateError = new(id: "VocaDbResXFileCodeGenerator001",
+		title: "Duplicate member",
+		messageFormat: "Ignored added member '{0}'",
+		category: "ResXFileCodeGenerator",
+		DiagnosticSeverity.Warning,
+		isEnabledByDefault: true);
+
+	private static readonly DiagnosticDescriptor MemberSameAsClassError = new(id: "VocaDbResXFileCodeGenerator002",
+		title: "Member same name as class",
+		messageFormat: "Ignored member '{0}' has same name as class",
+		category: "ResXFileCodeGenerator",
+		DiagnosticSeverity.Warning,
+		isEnabledByDefault: true);
```

Comment:
Can you please prefix private static fields with `s_`?

Reply:
done